ci.sh

```diff
@@ -21,6 +21,6 @@ python -m pip install -r test-requirements.txt
 
 #black --diff --check snekomatic
 
-pytest --cov="snekomatic" --cov="tests" --cov-report=term -r a --verbose
+pytest $*
```

Comment:
I think you mean `"$@"`? Not that we're using it either way, but hey, if you don't know about `"$@"` it's a good thing to know :-)

Reply:
Ah, right. Been in a hurry to show you how to solve your problem :)